tests/test_e2e/test_status.py

```diff
@@ -151,7 +153,10 @@ def _on_update_status(self, _):
     ),
 )
 def test_status_comparison(status):
-    entitystatus = _status_to_entitystatus(status)
-    assert entitystatus == entitystatus == status
-    assert isinstance(entitystatus, type(status))
-    assert repr(entitystatus) == repr(status)
+    if isinstance(status, UnknownStatus):
+        ops_status = ops.UnknownStatus()
+    else:
+        ops_status = getattr(ops, status.__class__.__name__)(status.message)
+    assert status == status == ops_status
```

Comment:
```suggestion
    assert status == ops_status
```

Reply:
This was there before (the names got reversed while I was trying something else out, but it's otherwise unchanged). I assume to verify that comparing to an object of the same type works properly. I think that's worth keeping. I could make it clearer that's what the test is doing.

Reply:
oh, it probably was a typo from when I removed the tuple-to-statusbase comparison
but it also makes sense as you say :)

Reply:
my 2c: either make it explicit or remove.

code is read more than written and I imagine every developer would consider `x == x == y` to likely be a typo.